lib/commands/implementations/banphrase.js

```diff
@@ -9,15 +9,32 @@ function banphrase(defaultPunishmentDuration, punishmentType) {
     const duration = _.get(matched, 1, '');
     const bannedPhrase = _.get(matched, 2, '').toLowerCase();
     let parsedDuration = defaultPunishmentDuration;
+
     if (duration !== '') {
       parsedDuration = parseDurationToSeconds(duration);
       if (parsedDuration === null) {
-        return new CommandOutput(
-          null,
-          'Could not parse the duration. Usage: "!AddX {amount}{m,h,d,w} {some banned phrase} " !AddMute 1d YOU BEEN GNOMED',
+        return new Promise((resolve) =>
```

Comment:
Would you mind using `Promise.resolve(value)`? It's a little more concise than `new Promise((resolve) => resolve(value)`.